src/models/index.js

```diff
@@ -38,8 +38,6 @@ module.exports.initSequelize = sequelize => {
 }
 
 const sequelizeConfig = {
-  // See http://docs.sequelizejs.com/manual/querying.html#operators-security
```

Comment:
Can you clarify why this change was necessary?

Reply:
For security purposes, string operators (`$or`, `$and`, etc) have been deprecated and `sequelize` will now show deprecation warnings, even though we're not using string operators anywhere.
I was reading through https://github.com/sequelize/sequelize/issues/8417 and it seems like the general consensus was to set `operatorsAliases` to `false`, but that will also trigger a warning as seen [here](https://github.com/sequelize/sequelize/blob/master/lib/sequelize.js#L244-L249).
Removing the option altogether has the same effect as setting it to `false`, but without the warning.